.drone.star

```diff
@@ -2801,3 +2773,137 @@ def setupForLitmus():
             "cat .env",
         ],
     }]
+
+def performWebCache(ctx):
+    ocis_git_base_url = "https://raw.githubusercontent.com/owncloud/ocis"
+    path_to_drone_env = "%s/%s/.drone.env" % (ocis_git_base_url, ctx.build.commit)
+    path_to_check_script = "%s/%s/tests/config/drone/check_web_cache.sh" % (ocis_git_base_url, ctx.build.commit)
+
+    return [
+        {
+            "name": "get-drone-env-and-check-script",
+            "image": OC_UBUNTU,
+            "commands": [
+                "curl -s -o .drone.env %s" % path_to_drone_env,
```

Comment:
that file and also the script is in your clone, why downloading it again?

Reply:
for this pipeline we just need two files from ocis repo: 
1. `.drone.env` and 
2. `script to check for existing cache`

For only these two filea, the whole repo clone is not necessary. So repo clone is disabled in this pipeline. And these necessary files are downloaded using the `rawcontent` facility of github where we can grab file content from specific commit id